common/src/cellular_pktio.c

```diff
@@ -676,7 +676,7 @@ static CellularPktStatus_t _handleMsgType( CellularContext_t * pContext,
         if( *ppAtResp == NULL )
         {
             *ppAtResp = _Cellular_AtResponseNew();
-            LogDebug( ( "Allocat at response %p", *ppAtResp ) );
+            LogDebug( ( "Allocat at response %p", ( void * ) *ppAtResp ) );
```

Comment:
Spelling error

Reply:
Thanks for the correction.

Reply:
Please refer to [this change](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/37/commits/86df4964329f419781572f4d629b65f6f5ec9a52) for the correction.